docs/articles/workflow-health.mdx

```diff
@@ -0,0 +1,61 @@
+# Workflow Health & Flakiness
+
+Testkube introduces **Workflow Health** as a core metric to help you evaluate the *reliability* and *stability* of your Test Workflows over time.
+This metric incorporates not only how often your workflows pass or fail, but also how consistent their outcomes are—an essential signal for identifying **flaky tests** and improving test reliability at scale.
+
+---
+
+## What is Test Flakiness?
+
+Test **flakiness** refers to non-deterministic or inconsistent test results—tests that sometimes pass and sometimes fail without changes to the underlying system. Flaky tests can lead to:
+
+- False negatives that block deployments
+- Wasted debugging time
+- Reduced developer confidence in test outcomes
+
+Testkube tracks flakiness across entire **workflows**, not just individual test runs, to provide a broader view of system reliability.
```

Comment:
we don't track individual test runs at all (yet) - suggest to put this in a note-block - and rephrase to 

"Testkube tracks flakiness at the Workflow-level to provide a borader view of system reliability. Test-level flakiness will be introduced in a future release"